src/components/MenuItem.js

```diff
@@ -137,6 +140,21 @@ const MenuItem = React.forwardRef((props, ref) => {
 
     const hasPressableRightComponent = props.iconRight || (props.rightComponent && props.shouldShowRightComponent);
 
+    const renderTitleContent = () => {
+        if (props.titleWithTooltips && _.isArray(props.titleWithTooltips) && props.titleWithTooltips.length > 0 && !props.shouldUseFullTitle) {
+            return _.map(props.titleWithTooltips, (accountIdOrUserObject, index) => (
```

Comment:
`accountIdOrUserObject` seems incorrectly named, aren't we only expecting it to be an object?